client/gutenberg/editor/style.scss

```diff
@@ -108,28 +100,7 @@ $gutenberg-theme-toggle: #11a0d2;
 			text-decoration: none;
 			top: 5px;
 			width: auto;
-			z-index: 100000; /* Above WP toolbar. */
-		}
-	}
-
-	.editor-block-list__block {
-		ul ul, ol ol {
-			list-style-type: circle;
+			z-index: 100000; // Above WP toolbar.
 		}
 	}
 }
-
-//needed for oembed iframes to appear
-.wp-block-embed__wrapper > iframe {
```

Comment:
Do we still need this, because `box-sizing` updates don't seem related?

Reply:
I actually tested this by following the instructions included in #27486 and I couldn't see any problems with the Oembed block.
@gwwar do you happen to remember if there are corner cases to check here?

Reply:
I went through the original testing steps too and confirmed that it works as expected even without these styles. 👍Might be the case that package updates made this fix obsolete.